src/uv.cc

```diff
@@ -112,8 +103,6 @@ void Initialize(Local<Object> target,
                   ->GetFunction(env->context())
                   .ToLocalChecked()).Check();
 
-  // TODO(joyeecheung): This should be deprecated in user land in favor of
```

Comment:
Why is the TODO being removed? It seems to be referencing something other than what's being changed in this PR?

Reply:
Nope, it's making reference to the use of `process.binding('uv').errname()` which is deprecated and should be using `util.getSystemErrorName(err)`

Reply:
This is actually talking about `process.binding('uv').UV_*` constants, not `process.binding('uv').errname()`.

Reply:
Sorry, my bad; I think I should close this.

But amm, I could help with the `process.binding('uv').UV_*` constants instead

Reply:
The behavior change here is moving `process.binding('uv').errname()` from `--pending-deprecation` to unconditional deprecation which is still a TODO so I don't think this needs to be closed. But this comment is not talking about that so it should not be removed.

Reply:
Great, thanks Joyee; I'll delete the changes of the comment.

Let me know if you want me to do the uv_const deprecations, was thinking about using `SetAccessor` on the for loop for emit the warning when the uv consts are called.